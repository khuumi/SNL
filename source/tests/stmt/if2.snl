if y > 15.9
(y is y + 7
z is z - 53.8)
else
(local x is false)
