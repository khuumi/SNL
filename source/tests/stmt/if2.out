If (Binop (Id Global y) Gt (Float 15.9)) (Block [(Assign (Id Global y) (Binop (Id Global y) Add (Int 7))),
(Assign (Id Global z) (Binop (Id Global z) Sub (Float 53.8)))]) (Block [(Assign (Id Local x) (Bool false))])